packages/docs/src/components/Layout.tsx

```diff
@@ -68,7 +90,7 @@ const Layout = ({
       <div className="ds-l-row ds-u-margin--0">
         <Navigation location={location} />
         <main id="main" className="ds-l-md-col ds-u-padding--0 ds-u-padding-bottom--4">
-          <header className="ds-u-padding--3 ds-u-sm-padding--6 ds-u-display--block ds-u-fill--gray-lightest">
+          <header className=" ds-u-border-bottom--1 ds-u-padding--3 ds-u-sm-padding--6 ds-u-display--block ds-u-fill--gray-lightest">
```

Comment:
Where is this coming from?

Reply:
It used to be on the <article> seen [here](https://github.com/CMSgov/design-system/pull/1949/files/692ee261dee4aa4ab0e26162f0880e3cb7984672#diff-5f64bcbb7029ef2d0ba4b6412b66385f968db76943a6f1564c80ed81cd15ebf5L103), But because of the way padding/margin was defined on the article, the border wasn't stretching full-width

Reply:
Let's remove both of these classes from the `<header>` : `ds-u-border-bottom--1`  and `ds-u-fill--gray-lightest`